easysid.cpp

```diff
@@ -38,6 +61,40 @@ EXPORT void end_capture()
     }
 }
 
+EXPORT void set_chip(uint32_t chip)
+{
+    if (chip < 0 || chip > 1) {
+        fprintf(stderr, "Invalid SID Chip: %d\n", chip);
+        // TODO: Implement error checking...
+        return;
+    }
+    sid.set_chip_model(static_cast<chip_model>(chip));
+}
+
+EXPORT void set_sampling_parameters(double clock, uint32_t sample_mode, uint32_t sample_freq)
```

Comment:
They are defined the yet-to-be-pushed new sid.neon module.  I realize that doesn't help EasySID.DLL users, but...  I will export the two MAIN clock variables, but the key is, you *CAN* vary the clock input frequency to mess with the SID's output sound...